tardis/io/tests/test_model_reader.py

```diff
@@ -0,0 +1,35 @@
+import tardis
+from tardis.io.model_reader import (read_artis_density,
+read_simple_ascii_abundances)
+
+from astropy import units as u
+import numpy as np
+
+import pytest
+
+import os
+
```

Comment:
System imports like os, sys should go to the top. Then leave a line followed by our tardis imports. Same for other files.
Read this https://github.com/Khan/style-guides/blob/master/style/python.md